.repo-metadata.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "django-google-spanner",
+  "name_pretty": "Cloud Spanner Django",
+  "product_documentation": "https://cloud.google.com/spanner/docs/",
+  "client_documentation": "https://googleapis.dev/django/spanner/latest",
+  "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:190851%2B%20status:open",
+  "release_level": "ga",
+  "language": "python",
+  "library_type": "GAPIC_COMBO",
```

Per go/library-types:
```suggestion
  "library_type": "INTEGRATION",
```